database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -22,6 +22,7 @@ public function up()
             $table->timestamp('mobile_verified_at')->nullable();
             $table->string('mobile_verify_code')->nullable();
             $table->tinyInteger('mobile_attempts_left')->default(0);
+            $table->timestamp('mobile_last_send');
```

Comment:
@MaxRomagnoli I would probably name the field `mobile_verify_code_sent_at` for clarity